app/web/src/workers/webworker.ts

```diff
@@ -307,8 +308,10 @@ const exec = (
 /**
  * A few small utilities
  */
+const textEncoder = new TextEncoder();
 const encodeDocumentForDB = async (doc: object) => {
-  return await new Blob([JSON.stringify(doc)]).arrayBuffer();
+  const docJson = JSON.stringify(doc);
+  return textEncoder.encode(docJson);
```

Comment:
we're fairly confident there isn't any functional difference here? And we see speed gains?

Reply:
i'm 100% confident there's no functional difference, but not sure where it caused a speed up or not, i'd have to measure again. it does mean we can remove the async form encodeDocumentForDB